src/ApparentHorizons/Tags.hpp

```diff
@@ -366,6 +366,25 @@ struct RicciScalarCompute : RicciScalar, db::ComputeTag {
   using return_type = Scalar<DataVector>;
 };
 
+/// Stores a YlmSpherepack object for computing gradients and intergrals on a
```

Comment:
If you add backticks around `YlmSpherepack` and `Strahlkorper`, they'll generate links in the documentation.

Reply:
`/// Stores a `YlmSpherepack` object for computing gradients and intergrals on a `Strahlkorper``

Reply:
Like this? @moxcodes